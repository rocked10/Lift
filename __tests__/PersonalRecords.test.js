import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import PersonalRecords from '../screens/personalRecords';

describe('Testing Personal Records Screen', () => {
    const navigate = jest.fn();
    const wrapper = renderer.create(<PersonalRecords navigation={navigate} route={{params: {_recordsToDisplay: ''}}} />);

    it('Should render', () => {
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});