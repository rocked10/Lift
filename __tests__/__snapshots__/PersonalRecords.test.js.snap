// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Personal Records Screen Should render 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 16,
    }
  }
>
  <Text
    style={
      Object {
        "fontFamily": "lato-regular",
        "fontSize": 18,
        "marginBottom": 16,
      }
    }
  >
    This is going on your profile! Choose what you want to show
  </Text>
  <RCTScrollView
    ItemSeparatorComponent={[Function]}
    data={null}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
